scripts/tools/zap/update_cluster_revisions.py

```diff
@@ -51,6 +70,34 @@ def getTargets(cluster_id: int):
     return targets
 
 
+def get_outdated_clusters(data: object, xml_clusters: dict, args) -> list[ClusterInfo]:
+    result = []
+    for endpoint in data.get("endpointTypes", []):
+        endpoint_id = endpoint.get("id") - 1
```

Comment:
That is NOT the endpoint_id. Endpoint IDs are separate from the `id` field. This may work oddly on ZAP files that have non-contiguous endpoint IDs.